src/js/contact.js

```diff
@@ -1,19 +1,4 @@
 export default function contact() {
-  const nav = document.createElement('nav');
-  nav.classList.add('nav');
-  const ul = document.createElement('ul');
-  ul.classList.add('ul');
-
-  const links = ['Home', 'Menu', 'Contact'];
-  for (let i = 0; i < 3; i += 1) {
-    const li = document.createElement('li');
-    li.innerHTML = links[i];
-    li.classList.add('li');
-    ul.appendChild(li);
-  }
-
-  nav.appendChild(ul);
-
   const mapouter = document.createElement('div');
   mapouter.classList.add('mapouter');
```

Comment:
- [ ] The following properties below are currently deprecated in iframes. Kindly remove them:
```
  iframe.frameBorder = '0';
  iframe.scrolling = 'no';
  iframe.marginHeight = '0';
  iframe.marginWidth = '0';
```